provisioner/ansible/provisioner.go

```diff
@@ -382,6 +391,21 @@ func validateFileConfig(name string, config string, req bool) error {
 	return nil
 }
 
+func validateDirectoryConfig(name string, config string, req bool) error {
```

Comment:
`config` never varies and is used in every error string created by `validateDirectoryConfig`. Is there any reason to keep the `config` parameter?